middlewares/allErrorsHandler.js

```diff
@@ -0,0 +1,10 @@
+const allErrorsHandler = (error, req, res, next) => {
```

Comment:
Can check for more errors here such as mongoose validation errors and other things to give frontend meaningful errors.